test/basic-querying.test.js

```diff
@@ -153,6 +153,27 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
```

Comment:
is it required for this test to search among these 4 users? unless i'm missing something could you simplify with only 2 different users?